src/getViteConfigPath.test.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, expect, it } from "vitest";
+import path from "path";
+import { getViteConfigPath } from "./getViteConfigPath";
+
+describe("getViteConfigPath", () => {
+  const mockWorkingDirectory = path.resolve(
+    __dirname,
+    "..",
+    "test",
+    "mockConfig"
+  );
+
+  it("resolves with a full path if a file at the provided path exists", async (): Promise<void> => {
+    await expect(
+      getViteConfigPath(mockWorkingDirectory, "vitest.config.all.js")
+    ).resolves.toMatch(/test\/mockConfig\/vitest\.config\.all\.js/i);
```

Comment:
Wouldn't a `.toContain('test/mockConfig/vitest.config.all.js')` work here as well?
That would read a little nicer - but I didn't test this, so if there is a good reason for `toMatch` that is perfectly valid as well! 🙂

Reply:
You're right, [`.toContain`](https://vitest.dev/api/expect.html#tocontain) would work just as well, in fact, [`.toMatch`](https://vitest.dev/api/expect.html#tomatch) also works with just a string parameter.
I initially went for the regex to get case-insensitive matching (which neither `.toContain` nor `.toMatch` provide with string parameters) but your comment made me realize that all the values I am trying to match on are defined in the test suite so there is no need.
So at this point I think it comes down to minor semantic differences between 'contain' and 'match'. I'm not leaning strongly in one way or another so if you have a preference I see no issue in switching 🙂

Reply:
`.toMatch` is fine in that case. Looks great! 👍🏻